common/metrics/nop.go

```diff
@@ -58,10 +64,38 @@ func (n noopClientImpl) UpdateGauge(scope int, gauge int, value float64) {
 }
 
 func (n noopClientImpl) Scope(scope int, tags ...Tag) Scope {
-	return NoopScope(Common)
+	return NoopScope
 }
 
 // NewNoopMetricsClient initialize new no-op metrics client
 func NewNoopMetricsClient() Client {
 	return &noopClientImpl{}
 }
+
+type noopScopeImpl struct{}
+
+func (n *noopScopeImpl) IncCounter(counter int) {
+}
+
+func (n *noopScopeImpl) AddCounter(counter int, delta int64) {
+}
+
+func (n *noopScopeImpl) StartTimer(timer int) Stopwatch {
+	return NewTestStopwatch()
```

Comment:
probably want the `NoopStopwatch` here instead, since you created it above.

but yea, a timer-less stopwatch here makes sense 👍 they don't do much.